test/component_cloudwatch_test.go

```diff
@@ -12,10 +12,10 @@ import (
 func Test_cloudwatch(t *testing.T) {
 	setup(t)
 
-	pkgTest.Boot("test_configs/config.cloudwatch.test.yml")
-	defer pkgTest.Shutdown()
+	mocks := pkgTest.Boot("test_configs/config.cloudwatch.test.yml")
+	defer mocks.Shutdown()
 
-	cwClient := pkgTest.ProvideCloudwatchClient("cloudwatch")
+	cwClient := mocks.ProvideClients("cloudwatch").CloudWatch
```

Comment:
This implementation looks strange. I liked the prior implementation more. As said: assertions are not a bad thing

Reply:
I'll refactor this as discussed.